pkgs/development/interpreters/yex-lang/derivation.nix

```
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, pkgconfig, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "yex-lang-${version}";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "yex-lang";
+    repo = "yex-lang";
+    rev = "a97def1431b73b8693700f530ec023f1776eaf83";
+    sha256 = "074x9j0ihjpaghnwywq5zyxfad2h6m57c2i58wkz6chma6vcjk08";
+    fetchSubmodules = true;
+  };
+  buildInputs = [  ];
+
+  checkPhase = "";
+  cargoSha256 = "sha256:0zx8h8mgaasivkmcq06c8ha4wafcfv6kgh0sb1k7g4k6q4j5ir3a";
+
+  meta = with lib; {
+    description = "cool language written in rust";
```

```suggestion
    description = "Cool language written in rust";
```

See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes